addons/l10n_au/data/account.tax.group.csv

```diff
@@ -1,4 +1,3 @@
-id,name
-tax_group_gst_0,GST 0%
-tax_group_gst_10,GST 10%
-tax_group_gst_100000000,GST 100000000%
```

Comment:
Who told you to remove this?

Reply:
You mean this? --> 
![immagine](https://user-images.githubusercontent.com/1665365/125634624-2cac1ae4-c781-4073-b508-dd48bac857ef.png)
It's coming from @tsb-odoo , @oco-odoo 

**Benj:** I've been discussing this with OCO
It is a false good idea
Our suggestion is to remove that rate of 1000000... %
in the standard AU localization, let's remove it.

**Paolo:** Aaaand... someone will complain :smile:

**Benj:** The user can still : 
- create its invoice 
- go to tab Journal Items
- set the tax grid manually
- post its invoice
But the logic of 1000000...% and tax price include is a hack, not something we want to promote in standard.

**Paolo:** ooook

Reply:
We need to ask them if the workaround works for them. Not just remove it without knowing.

Reply:
@richard-willdooit

Reply:
Other work around identified in the meantime. 
Tax with 0% amount
Only a base line in the tax repartition lines. No tax repartition
Tax grids specified on the Base Repartition Line
Then on the vendor bill, the account to be used is not an expense account but a tax paid account.

Reply:
Change moved to https://github.com/odoo/odoo/pull/73890, see task  https://www.odoo.com/web#id=2278230&model=project.task